### DISCUSSION TITLE
Adding a new monster flag to use meat for taming animals

### DISCUSSION THREAD

**Discussion by ghost** [cataclysmbn/Cataclysm-BN#2612] [General]
A number of animals use Meat as a placate trigger, adding a flag for raw meat to be used for taming animals in the same method as cattle fodder would allow mods to add taming wolves, foxes, possibly other ambitious ideas like cougars and bears. In the mainline perhaps wolves could be tamed, while mods would be able to make a lot of other animals too unbalanced or unrealistic such as the bear idea.

Alternate solution: A new entry within monster JSON to allow you to set any item or item group as a tame item, instead of flagging it "Cattlefodder" "Dogfood" or the hypothetical new "Rawmeat" flags. This would likely take a lot more work however.

**Comment by Zlorthishen** [comment]
pet food should just be jsonized

**Reply by chaosvolt** [reply]
That'd make it easier overall, yeah.
